### PR TITLE
d8 version of HSTS module

### PR DESCRIPTION
**Pull request by mfb**

### CODE REVIEW

**Comment by mfb** [comment]
Also got Travis CI working @ https://travis-ci.org/mfb/hsts

File: config/install/hsts.settings.yml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+max_age: 300
```
Comment (minorOffense) [inline config/install/hsts.settings.yml]: Can you set the default max age to 500 to match the default in 7.x please

File: hsts.permissions.yml [hunk 2]
```diff
@@ -0,0 +1,3 @@
+administer strict transport security:
```
Comment (minorOffense) [inline hsts.permissions.yml]: This is actually left over from my newbiness in the 6.x and 7.x versions. Can you change the permission machine name to be "administer hsts". That way we don't collide with other permissions / way easier to type when checking permissions.

File: hsts.routing.yml [hunk 3]
```diff
@@ -0,0 +1,7 @@
+hsts.settings:
+  path: '/admin/config/system/hsts'
+  defaults:
+    _form: 'Drupal\hsts\Form\HstsConfigForm'
+    _title: 'HSTS settings'
+  requirements:
+    _permission: 'administer strict transport security'
```
Comment (minorOffense) [inline hsts.routing.yml]: Change this along with the permission name above

**Comment by minorOffense** [comment]
Rest looks good. And super awesome job with the Travis CI stuff. Fix those few items and I'll merge it onto d.o too.

**Pull request merged**